gateway/api/views/jobs.py

```diff
@@ -51,10 +53,39 @@ class JobViewSet(viewsets.GenericViewSet):
 
     BASE_NAME = "jobs"
 
+    jobs_repository = JobsRepository()
+
     def get_serializer_class(self):
+        """
+        Returns the default serializer class for the view.
+        """
         return self.serializer_class
 
+    @staticmethod
+    def get_serializer_job(*args, **kwargs):
+        """
+        Returns a `JobSerializer` instance
+        """
+        return v1_serializers.JobSerializer(*args, **kwargs)
```

Comment:
Take a look how we are using the Serializers in the Program, @paaragon :
- In V1 we specify the V1 serializers: https://github.com/Qiskit/qiskit-serverless/blob/main/gateway/api/v1/views/programs.py#L25
- And in views we are using the default one: https://github.com/Qiskit/qiskit-serverless/blob/main/gateway/api/views/programs.py#L64

So here you should use the `JobSerializer` and this one in the v1 view.